### ISSUE TITLE
running in git repo subdirectory throwing error 'could not get git status --porcelain'

### ISSUE THREAD

**Issue by JustinBeaudry** [mikaelbr/gulp-gitmodified#5]
```
 [Error: Could not get git status --porcelain]
  plugin: 'gulp-gitmodified',
  showStack: false,
  name: 'Error',
  message: 'Could not get git status --porcelain',
  stack: 'Error: Could not get git status --porcelain\n    at /Users/justinbeaudry/cl/src/frontend/mcp/node_modules/gulp-gitmodified/lib/git.js:27:19\n    at ChildProcess.exithandler (child_process.js:758:5)\n    at ChildProcess.emit (events.js:110:17)\n    at maybeClose (child_process.js:1015:16)\n    at Socket.<anonymous> (child_process.js:1183:11)\n    at Socket.emit (events.js:107:17)\n    at Pipe.close (net.js:485:12)' }
Stacktrace:
Error: Could not get git status --porcelain
    at /Users/justinbeaudry/cl/src/frontend/mcp/node_modules/gulp-gitmodified/lib/git.js:27:19
    at ChildProcess.exithandler (child_process.js:758:5)
    at ChildProcess.emit (events.js:110:17)
    at maybeClose (child_process.js:1015:16)
    at Socket.<anonymous> (child_process.js:1183:11)
    at Socket.emit (events.js:107:17)
    at Pipe.close (net.js:485:12)
```

Simply running git status --porcelain in the subdirectory seems to work fine.

**Comment by mikaelbr** [comment]
Ah, yeah. So it runs git from your root directory no matter what, for now. We can add an option to specify git root path, though. As in:

```js
var files = gulp.src('./src/*.ext')
  .pipe(gitmodified({
     mode: 'modified',
     gitRoot: 'someSubDirectory/'
  }));
```

**Comment by mikaelbr** [comment]
Fixed and added in `v1.1.0`. See README for example.
